spec.md

```diff
@@ -89,67 +85,109 @@ If a license does not exist in the SPDX License List, its SPDX License Identifie
 MUST be `LicenseRef-[idstring]` as defined by the SPDX Specification, Clause 10 available at <https://spdx.github.io/spdx-spec/v2.3/other-licensing-information-detected/>.
 
 A Project MUST NOT include License Files for licenses under which none of the
-files in the Project are licensed.
+files in the Project are licensed. The `LICENSES/` directory MUST NOT include
+any other files.
 
 Everything that applies to licenses in this section also applies to license
 exceptions, with the exception that it is NOT possible to have a license
 exception that does not exist in the SPDX License List.
 
-For avoidance of doubt, in practice this means that for every license and exception
-that is part of any SPDX License Expression in any Copyright and Licensing Information
-associated with any Covered File, there MUST exist a License File as defined in this section.
+For avoidance of doubt, in practice this means that for every license and
+exception that is part of any SPDX License Expression in any Copyright and
+Licensing Information associated with any Covered File, there MUST exist a
+License File as defined in this section.
+
+You MAY include a `COPYING` or `LICENSE` file (with or without file extensions)
+in your project for compliance with other standards, conventions, or tools.
+These files MAY contain a copy of the license text, a summary of your licensing,
+or anything else. These files are ignored by the REUSE Tool.
 
 ## Copyright and Licensing Information
 
 Each Covered File MUST have Copyright and Licensing Information associated with
-it. There are two ways to associate Copyright and Licensing Information with a
-file. In addition, there is a way to associate Copyright and Licensing
-Information with a snippet.
+it. You can associate Copyright and Licensing Information with a file in the
+following ways:
+
+- Comment headers
+- REUSE.toml
+- DEP5 (deprecated)
+
+REUSE.toml and DEP5 are mutually exclusive. You MUST NOT use both
+simultaneously.
+
+Additionally, you can associate Copyright and Licensing Information with
+Snippets inside of files.
```

Comment:
Can we refer to the separate section, or mention it somehow (like "see below" or so)?

Reply:
sure, agree

Reply:
I am deciding against this because everything in this section is documented in its subsections. This text is just a small preamble, and the relevant section here is almost immediately below the referenced text.